apps/prairielearn/elements/pl-graph/pl-graph.py

```diff
@@ -193,7 +193,7 @@ def render(element_html: str, data: pl.QuestionData) -> str:
         if not log_warnings:
             warnings.simplefilter("ignore")
 
-        svg_bytes = translated_dotcode.draw(format="svg", prog=engine)
+        svg_bytes = translated_dotcode.draw(format="inline_svg", prog=engine)
```

Comment:
This doesn't work for me locally, have you tested it? As mentioned before, the svg_inline (not inline_svg) option is only available in newer versions of graphviz, which are not available on PL image or on the Ubuntu repo (which is the current installation suggestion for WSL).

Reply:
Ah sorry, I just tested it on my local environment, not in the PL image / Ubuntu repo. Will test that now.

Reply:
Seems like the homebrew version is newer than the Ubuntu/Amazon Linux versions.

Reply:
https://gitlab.com/graphviz/graphviz/-/merge_requests/4075

Reply:
Confirmed that the regex ([444cbd8](https://github.com/PrairieLearn/PrairieLearn/pull/11977/commits/444cbd88e262e5d31dfa0d1856cdb335b9c3d901[879da81](https://github.com/PrairieLearn/PrairieLearn/pull/11977/commits/879da81918a4d02f045317ada8f768ade128ef20)) and  at least works for me locally on my mac.
![CleanShot 2025-05-13 at 10 58 54@2x](https://github.com/user-attachments/assets/13100294-ed86-492d-aa4c-2d5770372ae6)

Reply:
Added some docs in f0f064a2a6830ed042a57c293ac544b023d20c79

Reply:
I [commented in the commit](https://github.com/PrairieLearn/PrairieLearn/commit/f0f064a2a6830ed042a57c293ac544b023d20c79#r156933711), but those can sometimes be lost, so repeating here:

> Nit: https://gitlab.com/graphviz/graphviz/-/merge_requests/3404 may be a better link here. AFAIK Amazon Linux is not affected by the Debian issues.

Reply:
Changed in 1d3119a0b0bcd30053f2346c888c8fe3e1dd516b.